### DISCUSSION TITLE
Static Volumes

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#801] [Questions] [answered]
Just a short question, for what are the static volumes "nextcloud_aio_onlyoffice", "nextcloud_aio_clamav" and "nextcloud_aio_apache" used?

**Comment by szaimen** [comment, marked as answer]
> nextcloud_aio_onlyoffice

Stores the files that you edit in onlyoffice. Useful if onlyoffice should fail saving them back to nextcloud.

> nextcloud_aio_clamav

Stores the AntiVirus database



> nextcloud_aio_apache

Stores your certificates
